auctioneerrpc/auctioneer.proto

```diff
@@ -965,6 +965,12 @@ message ServerBid {
     correspond to the recipient node's details.
     */
     bool is_sidecar_channel = 7;
+
+    /*
+    Flag used to decided if the channels resulting from this order's matches
+    should be announced to the network or not.
+    */
+    bool private_channel = 8;
```

Comment:
Perhaps we should rename this to "unadvertised"?

Reply:
Or even move it ot the `ServerOrder` field instead. 

I think one other question here is how we want to implement matching:
  * Should we let only the bid specify if the channel is private or not? 
  * Or should we also give the ask side the freedom to express if they want to accept an unadvertised channel or not?

Reply:
>should we also give the ask side the freedom to express if they want to accept an unadvertised channel 
Should this read: "accept **an order for** an unadvertized..."

Seems the domains for decisions here are:
{bid,ask} {exclusively unannounced channel, ambivalent}

Reply:
discussed offline with @dstadulis, new field for asks added.